### PR TITLE
Pipeline, Lopatko Yulia - 21FPL1

### PR DESCRIPTION
**Pull request by lazykosti**

### CODE REVIEW

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Good start for pipeline. Please continue work

**Comment by dmitry-uraev** [comment]
No progress since last review

**Comment by dmitry-uraev** [comment]
No progress since last review

**Review by dmitry-uraev — APPROVED** [review]

**Comment by dmitry-uraev** [comment]
You are done for score 4 now

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -1,129 +1,237 @@
...
         Converts the Mystem tags into the UD format
         """
+        pos = self.convert_pos(tags)
+        pos_feats = {
+            'NOUN': [self.case, self.number, self.gender, self.animacy],
+            'VERB': [self.tense, self.number, self.gender],
+            'ADJ': [self.case, self.number, self.gender],
+            'NUM': [self.case, self.number, self.gender],
+            'PRON': [self.case, self.number, self.gender, self.animacy]
+        }
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pipeline.py:208]: is it necessary to define pos_feats structure like this explicitly?

**Review by dmitry-uraev — COMMENTED** [review]
Good job! Let's pass CI checks

File: lab_6_pipeline/pipeline.py (hunk 1 above)
Comment (lazykosti) [reply lab_6_pipeline/pipeline.py:208]: nope! I give up anyway. So I guess I'm going to close PR now
Thank you for 4 :)

**Pull request closed without merging**